Picture.scala

```diff
@@ -0,0 +1,159 @@
+package pioneer.pictures
+
+import java.awt.Color
+import java.awt.image.BufferedImage
+import java.io._
+import javax.imageio.ImageIO
+
+/**
+  * A small, image-processing library. Thanks to
+  *    http://otfried.org/scala/image.html
+  * for help on how to perform basic image processing in Scala
+  */
+object Picture {
+
+  /** Flips an image along its horizontal axis */
+  def flipHorizontal(img: BufferedImage): BufferedImage = {
+    val image = img
```

Comment:
This line works, but it isn't strictly necessary: You could name the parameter `image` and remove this line.